content/history/intro.inc

```diff
@@ -2,4 +2,4 @@
 
 <p>One of the Buggy Alumni Association's primary goals is to act as a repository for buggy history. Check out the links above to explore the historical information we have collected.</p>
 
-<p>We are hoping to finish a system to allow efficient contributions of new data and corrections sometime soon. In the mean time, if you have something important to share, send an email to <a href="mailto:admin@cmubuggy.org">admin@cmubuggy.org</a>.</p>
+<p>We know our data is incomplete and sometimes inaccurate.  If you have something important to share, send an email to <a href="mailto:admin@cmubuggy.org">admin@cmubuggy.org</a>.</p>
```

Comment:
Thank you for updating! Could we please update the email to data@cmubuggy.org? (This includes admin@, dev@, Bryan, and Jeremy)

Reply:
Done.